app/helpers/application_helper.rb

```diff
@@ -7,4 +7,8 @@ def user_image_tag(owner)
     email = owner.nil? ? nil : owner.email
     gravatar_image_tag(email)
   end
+
+  def any_minion?
+    !Minion.all.count.zero?
```

Comment:
Didn't know about `any?` on model. I always prefer to expose a helper/variable instead of classes on templates but that's my preference. 

It would be nice to have explicit code standards for stuff that is not automated linted. I know we have rubocop but it doesn't analyze template files. I found https://github.com/flyerhzm/rails_best_practices, might help with some tweaks? Need to investigate.